### ISSUE TITLE
"csresnext50-panet-spp-original-optimal" model FPS 

### ISSUE THREAD

**Issue by mnbv7581** [AlexeyAB/darknet#5122]
We combined our custom dataset with the COCO dataset to learn. Target models used the two versions of [csresnext50-spp-original-optimal] and [yolo-spp] and unified the scale to 608x608. The problem here is that [csresnext50-spanet-spp-original-optimal] models show faster and more accurate performance than [yolo-spp] models, but the mAP shows better performance, but the speed is rather slow and confusing. Is there a possibility that I didn't learn well?

> **yolo-spp**
[net]
batch=1
subdivisions=1
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
learning_rate=0.00025
burn_in=1000
max_batches = 800200
policy=steps
steps=400000,500000,600000,700000
scales=.1,.1,.1,.1
[yolo]
mask = 0,1,2
anchors = 15, 29,  35, 70,  46,162, 100,102,  82,265, 209,199, 139,385, 262,440, 469,485
classes=5
num=9
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
iou_normalizer=1.25
iou_loss=giou

-------------------------------------------------------------------------------------------------------
> **csresnext50-panet-spp-original-optimal**
[net]
batch=1
subdivisions=1
width=608
height=608
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
learning_rate = 0.0006525
burn_in=4000
max_batches = 2002000
policy=steps
steps=400000,450000
scales=.1,.1
[yolo]
mask = 6,7,8
anchors = 15, 29,  35, 70,  46,162, 100,102,  82,265, 209,199, 139,385, 262,440, 469,485
classes=5
num=9
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
scale_x_y = 1.05
iou_thresh=0.213
cls_normalizer=1.0
iou_normalizer=0.07
uc_normalizer=0.07
iou_loss=ciou
nms_kind=greedynms
beta_nms=0.6
beta1=0.6

-----------------------------------------------------------------------------------------------------------

**Comment by WongKinYiu** [comment]
it seems group convolution is not efficient in modern GPU, i suggest you to use CSPDarknet53-PANet-SPP instead of CSPResNeXt50-PANet-SPP in this case.

#

**Detector FPS on GeForce RTX 2070 (Tensor Cores):**

`FPS` - measured using the command: `./darknet detector demo cfg/coco.data ... -benchmark`
`CUDNN_HALF=1` (Mixed-precision is forced for Tensor Cores (if groups==1))

1. **512x512:**
`yolov3-spp` - **52.0 FPS** - (--ms )
`csresnext50-panet-spp` - **36.5 FPS** - (--ms )

2. **608x608:**
`yolov3-spp` - **38.0 FPS** - (--ms )
`csresnext50-panet-spp` - **33.9 FPS** (--ms )

# 

`CUDNN_HALF=0`

1. **512x512**:
`yolov3-spp` - **41.4 FPS** - (--ms)
`csresnext50-panet-spp` - **34.5 FPS** - (--ms)

2. **608x608**:
`yolov3-spp` - **26.1 FPS** - (--ms)
`csresnext50-panet-spp` - **30.0 FPS** (--ms)

[more details](https://github.com/AlexeyAB/darknet/issues/4406#issue-530052808)
